mobile-ui/src/main/java/org/sagebionetworks/research/mobile_ui/show_step/view/ShowStepFragmentBase.java

```diff
@@ -75,28 +72,25 @@
  *         The type of StepViewModel that this fragment uses.
  */
 public abstract class ShowStepFragmentBase<StepT extends StepView, ViewModelT extends ShowStepViewModel<StepT>,
-        StepViewBindingT extends StepViewBinding<StepT>> extends ShowStepFragment {
+        StepViewBindingT extends StepViewBinding<StepT>> extends Fragment {
```

Comment:
In my next pr this should be extending ShowStepFragment instead of fragment. I refactored ShowStepFragment to be the base for all Fragments we use, and changed what was the old ShowStepFragment to be called ShowGenericStepFragment.

Reply:
I think you refactored it to have setPerformTaskFragment but not inherit the ShowStepFragmentBase stuff. PerformTaskFragment is now back to being injected with corrected scoping so is there still a need for ShowStepFragment? My thought is the Tracking fragment just needs to inject PerformTask

Reply:
Yeah as long as the PerformTaskFragment injects with the correct scope I don't believe we do need ShowStepFragment.